katas/metal-slug/jeanpeare/Character.js

```diff
@@ -0,0 +1,61 @@
+export class Character {
+    constructor (CharacterType, Health) {
+        this.CharacterType = CharacterType;
```

Comment:
This attribute makes sense, however, if you are planning to add subclasses for this class, at some point the type should be defined by the subclass itself instead of an attribute of it. Consider that for next iterations